cmake/macos/SignBundle.cmake

```diff
@@ -0,0 +1 @@
+execute_process(COMMAND codesign --deep -fs - ${CPACK_TEMPORARY_INSTALL_DIRECTORY}/vpkedit.app)
```

Comment:
Is it actually called `vpkedit.app` and not `VPKEdit.app`? macOS has a case-insenstive filesystem but this should use the correct case (and appropriate cmake variable)